### PR TITLE
red cross clarifications

### PR DESCRIPTION
**Pull request by danbjoseph**

### CODE REVIEW

**Comment by mikelmaron** [comment]
@danbjoseph can you explain why we need three bullets for ICRC, IFRC, and national societies? Nice to see a fuller list of national societies. Is there anyway to collapse the initial 3 bullets back down to one?

File: about-osm-community/_posts/0000-01-04-consumers.md [hunk 1]
```diff
@@ -62,8 +62,10 @@ Here is a non-exhaustive list of some big organizations that use OpenStreetMap,
 * Clinton Health Access Initiative
 * Concern Worldwide
 * Gates Foundation
-* International Red Cross Red Crescent Societies (IFRC) and the International Committee of Red Cross (ICRC)
-  * e.g. American Red Cross, British Red Cross, Canadian Red Cross, International Federation of Red Cross Red Crescent Societies, Netherlands Red Cross and more
```
Comment (danbjoseph) [inline about-osm-community/_posts/0000-01-04-consumers.md]: "International Federation of Red Cross Red Crescent Societies" this is not a thing

File: about-osm-community/_posts/0000-01-04-consumers.md [hunk 2]
```diff
@@ -62,8 +62,10 @@ Here is a non-exhaustive list of some big organizations that use OpenStreetMap,
 * Clinton Health Access Initiative
 * Concern Worldwide
 * Gates Foundation
-* International Red Cross Red Crescent Societies (IFRC) and the International Committee of Red Cross (ICRC)
```
Comment (danbjoseph) [inline about-osm-community/_posts/0000-01-04-consumers.md]: "International Red Cross Red Crescent Societies" is not the correct spelled out for IFRC

**Comment by danbjoseph** [comment]
the "International Red Cross and Red Crescent Movement" consists of the ICRC, the IFRC, and all the National Societies. all are independent bodies with their own individual status and no authority over the others.

for one bullet point could do something like:
* International Federation of Red Cross and Red Crescent Societies (IFRC), International Committee of Red Cross (ICRC), and National Societies (e.g. American Red Cross, British Red Cross, Canadian Red Cross, Netherlands Red Cross, and more)

The National Societies are not examples of the IFRC and ICRC as suggested in the existing text.

**Comment by mikelmaron** [comment]
Can we just say something in one bullet like the "International Red Cross and Red Crescent Movement", and list some of the key active national societies?

**Comment by dalekunce** [comment]
@mikelmaron I get that this is wordy but its really important to us Red Cross folks. Its akin to saying something like "Open Street Maps" for us. I think @danbjoseph is spot on with the revised bullet.

**Comment by mikelmaron** [comment]
Let's go with the single bullet. Can you make that change @danbjoseph

**Pull request merged**